main.go

```diff
@@ -21,8 +22,19 @@ func genLog() gol.Logger {
 }
 
 func main() {
+	dir, err := os.Getwd()
+	if err != nil {
+		log.Fatal(err)
+	}
+	// Best effort attempt, don't want to interfere with the logic
+	// below, I'll let the maintainer decide how to use this
+	// helper.
```

Comment:
Thank you so much for this approach! I've just opened a pull request to your fork to show you a way to use this helper. Hope you like it.